csr/csr.go

```diff
@@ -84,7 +84,7 @@ func VerifyCSR(csr *x509.CertificateRequest, maxNames int, keyPolicy *goodkey.Ke
 		}
 	}
 	if len(badNames) > 0 {
-		return fmt.Errorf("policy forbids issuing for: %s", strings.Join(badNames, ", "))
+		return fmt.Errorf("policy forbids issuing for: %q", strings.Join(badNames, ", "))
```

Comment:
It seems as though if there are multiple `badNames` that this will produce something like:

`"policy forbids issuing for: "google.cpu, microsoft.cpu, apple.cpu"` when I think we want:
`"policy forbids issuing for: "google.cpu", "microsoft.cpu", "apple.cpu"` such that if any individual `badName` has a `"` it would show up clearly.

Do you think its worth quoting the `badNames` individually and leaving the `%s` in the `fmt.Errorf`?